<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UC_Propreties_Navigation</class>
 <widget class="QFrame" name="UC_Propreties_Navigation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="Propreties_Navigator">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>800</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_lands">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color:#e2d82d;
font-size: 40px;</string>
    </property>
    <property name="text">
     <string>الاراضى</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/land-sales.png</normaloff>imgs/land-sales.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sakani">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>160</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color:#d81a1a;
font-size: 40px;
</string>
    </property>
    <property name="text">
     <string>السكنى</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/building.png</normaloff>imgs/building.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5adami">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color:#4689e6;
font-size: 40px;</string>
    </property>
    <property name="text">
     <string>الخدمى </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/museum.png</normaloff>imgs/museum.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sena3i">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color:#61d309;
font-size: 40px;</string>
    </property>
    <property name="text">
     <string>الصناعى </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imgs/factory2.png</normaloff>imgs/factory2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
